<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QDialog" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="frmUpdates">
     <property name="title">
      <string>Software Updates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkUpdates">
        <property name="text">
         <string>Check for software updates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDevelUpdates">
        <property name="text">
         <string>Include development (beta) versions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btns">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btns</sender>
   <signal>accepted()</signal>
   <receiver>SettingsWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btns</sender>
   <signal>rejected()</signal>
   <receiver>SettingsWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUpdates</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkDevelUpdates</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
